tap_github/repository_streams.py

```diff
@@ -882,6 +882,10 @@ class IssueCommentsStream(GitHubRestStream):
     # we have gaps in our data
     tolerated_http_errors = [502]
 
+    # GitHub is not missing the "since" parameter on this endpoint.
+    # But it is too expensive on large repos and results in a lot of server errors.
+    missing_since_parameter = True
```

Comment:
I find this name a bit confusing. Should it be called something like `use_fake_since_parameter`? As your comment says, the `since` parameter is not actually missing.

Reply:
I see what you were doing. Makes sense. I was thinking of having 2 different vars, like `missing_since_parameter` and `use_fake_since_parameter` but then the code would be super ugly. I think this is fine. Nothing that a good comment cannot fix :)